pkg/daemon/ceph/client/upgrade.go

```diff
@@ -160,12 +163,17 @@ func OkToStop(context *clusterd.Context, clusterInfo *ClusterInfo, deployment, d
 		if osdDoNothing(context, clusterInfo) {
 			return nil
 		}
+		maxRetries = 1
```

Comment:
With maxRetries=1, I believe we will sleep the whole time before checking again. Instead, we want a higher number of retries so we can check periodically before the whole timeout expired. For example, can we calculate a maxRetries that will cause a retry every 5-10s?